### PR TITLE
Add loop for filesystem check and move it to the create phase

### PR DESCRIPTION
**Pull request by johscheuer**
I added the points 2-3 from this issue https://github.com/quobyte/docker-volume/issues/21 

Any thoughts @quolix @LLParse ?

### CODE REVIEW

File: quobyte_driver.go [hunk 1]
```diff
@@ -72,16 +79,30 @@ func (driver quobyteDriver) Remove(request volume.Request) volume.Response {
 	return volume.Response{Err: ""}
 }
 
+func check_mount_point(mPoint string) error {
```
Comment (LLParse) [inline quobyte_driver.go]: Minor nitpick - can we make this camelCase, where the first letter is upper case iff the function should be exported?

File: quobyte_driver.go [hunk 2]
```diff
@@ -72,16 +79,30 @@ func (driver quobyteDriver) Remove(request volume.Request) volume.Response {
...
+	// We try it 5 times ~5 seconds -> move this into config?
+	max_tries := 5
+	tries := 0
+	var mount_error error
+	for tries <= max_tries {
```
Comment (LLParse) [inline quobyte_driver.go]: This looks like what we want - I am not familiar enough with Quobyte to know if there is any sort of SLA for newly created volumes propagating to client mountpoints. Maybe max_tries should be exposed as a flag until this is better understood? Also, if you want to get really fancy, exponential backoff wouldn't be a bad idea:
https://github.com/rancher/windows-scheduler/blob/master/metadata.go#L77-L96

**Comment by LLParse** [comment]
Thanks for this @johscheuer, I will test it out when I get a chance.

File: quobyte_driver.go (hunk 1 above)
Comment (johscheuer) [reply quobyte_driver.go]: Thanks, my fault to much Python the last weeks :)

**Comment by LLParse** [comment]
LGTM

File: quobyte_driver.go [hunk 3]
```diff
@@ -72,16 +79,30 @@ func (driver quobyteDriver) Remove(request volume.Request) volume.Response {
...
+		mount_error = nil
+		if fi, err := os.Lstat(mPoint); err != nil || !fi.IsDir() {
```
Comment (quolix) [inline quobyte_driver.go]: @quobert Please help on how to detect a Quobyte mount point more reliably.

**Pull request merged**